index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portafolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <div class="logo-container">
+        <a href="index.html" class="logo">My Logo</a>
+      </div>
+      <nav class="navbar">
+        <ul class="navbar-menu">
+          <li><a href="#">Portafolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <button type="button" class="btn-mobile-menu">
+        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" clip-rule="evenodd" d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z" fill="#6070FF"/>
+        </svg>
+      </button>
+    </header>
+    <main>
+      <section class="headline-section">
+        <div class="section-mobile-bg"></div>
+        <div class="section-mobile-shape"></div>
+          <div class="container">
+            <h1 class="main-greeting">
+              <span>I’m Nitcelis.</span>
+              <span>Glad to see you!</span>
+            </h1>
+            <p class="main-message">
+              I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <div class="social-media">
+              <p class="connect-message">LET’S CONNECT</p>
+              <ul class="social-media-links">
+                <li><a href="#"><img src="./img/github-icon.png" alt="github icon"></a></li>
+                <li><a href="#"><img src="./img/linkedin-icon.png" alt="linkedin icon"></a></li>
+                <li><a href="#"><img src="./img/angelist-icon.png" alt="angelist icon"></a></li>
+                <li><a href="#"><img src="./img/twitter-icon.png" alt="twitter icon"></a></li>
+                <li><a href="#"><img src="./img/medium-icon.png" alt="medium icon"></a></li>
+              </ul>
```

Comment:
- [OPTIONAL] Just as the previous reviewer said, you did a great job using the `ul` and the `a` tag here. I'll suggest you add the actual `href` links to your social media profiles here. It will make it easier for people to browse through your projects and reach about to you. :taco: :+1: :zap:

Reply:
Hi, @V-Blaze Thank you so much for those suggestions, I'll take them into account to improve my project and my professional skills.